### PR TITLE
[CRN-981] Title attribute hover

### PR DESCRIPTION
**Pull request by ricardocarvalh0**

### CODE REVIEW

File: apps/storybook/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "node": "16.x"
   },
   "scripts": {
-    "start": "start-storybook --quiet -c config -p 3001",
+    "start": "start-storybook --quiet --no-open -c config -p 3001",
```
Comment (kmaid) [inline apps/storybook/package.json:9]: ?

Comment (kmaid) [reply apps/storybook/package.json:9]: You should add STORYBOOK_NO_OPEN=true to your .env file

File: packages/react-components/src/atoms/__tests__/Link.test.tsx [hunk 2]
```diff
@@ -199,3 +199,56 @@ describe('when button-styled', () => {
     expect(getByText('text').closest('a')).not.toHaveAttribute('href');
   });
 });
+
+describe('for elipsed links', () => {
+  // !! Unfortunately jest does not respect ellipsed texts (overflow: hidden)
+  // https://github.com/testing-library/react-testing-library/issues/751#issuecomment-691069646
+  // so we cannot test if the text is shown breaked as it should
+  // but we can test other properties and see it working in the StoryBook
```
Comment (kmaid) [inline packages/react-components/src/atoms/__tests__/Link.test.tsx:207]: We do have playwright to test things like this

File: packages/react-components/src/atoms/__tests__/Link.test.tsx [hunk 3]
```diff
@@ -199,3 +199,56 @@ describe('when button-styled', () => {
...
+        'A very long text showing that the elipsis feature is working well, trying to provide as much text here as I can to validate this feature.';
+      const { getByTitle } = render(
+        <div css={{ maxWidth: '10px' }}>
+          <Link href="/" ellipsed>
+            {longText}
+          </Link>
+        </div>,
+      );
+
+      expect(getByTitle(longText)).toBeInTheDocument();
```
Comment (kmaid) [inline packages/react-components/src/atoms/__tests__/Link.test.tsx:221]: This is already tested. You should be using .closest('a').toHaveAttribute('title', 'expected string')

File: apps/storybook/package.json (hunk 1 above)
Comment (ricardocarvalh0) [reply apps/storybook/package.json:9]: Sorry for that, I forgot to remove before commit.

File: packages/react-components/src/atoms/__tests__/Link.test.tsx (hunk 2 above)
Comment (ricardocarvalh0) [reply packages/react-components/src/atoms/__tests__/Link.test.tsx:207]: For now we don't have any page using this Link component. I guess the components using it are going to be implemented in the future and then we'll be able to introduce the `e2e` tests. But anyways I can remove these comments.

Comment (ricardocarvalh0) [reply packages/react-components/src/atoms/__tests__/Link.test.tsx]: I've talked with João and he has provided me a page where we can use it for now. Will do it.

File: packages/react-components/src/atoms/Link.tsx [hunk 4]
```diff
@@ -47,17 +48,23 @@ const iconThemeStyles: Record<ThemeVariant, SerializedStyles> = {
   }),
 };
 
+const elipsedTextStyle = css({
```
Comment (kmaid) [inline packages/react-components/src/atoms/Link.tsx:51]: Could these styles be imported from the Ellipsis component?

**Review by kmaid — APPROVED** [review]

**Pull request merged**